.gitignore

```diff
@@ -0,0 +1,7 @@
+# Python # 
+*.py[cod] 
```

Comment:
What does this `*.py[cod]` mean here?

Reply:
basically this ignore any py file that ends with c,o,d .. means ignoring pyc,pyo,pyd files which are not necessry for any VS. these modules are automatically downloaded or added during compilation.